src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
```

Comment:
There is no need to create 3 instances of a ```Lottery``` class because the value returned by a ```getRandomBall``` method will be random without any connection to an instance of a ```Lottery``` class is used, so please replace creation of 3 instances of this class by creation of one, and invoke it three times to get 3 random balls